2024-04-16 21:18:10 UTC
Windows-10-10.0.22631
Ren'Py 8.2.1.24030407

Early init took 0.07s
Loading error handling took 0.05s
Loading script took 1.02s
Loading save slot metadata took 0.03s
Loading persistent took 0.00s
Running init code took 0.07s
Loading analysis data took 0.00s
Analyze and compile ATL took 0.01s
Reloading save slot metadata took 0.02s
Index archives took 0.00s
Dump and make backups took 0.07s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc took 0.02s
DPI scale factor: 1.500000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: b"Couldn't load nvlib." (can be ignored)
Creating interface object took 0.00s
Cleaning stores took 0.00s
Init translation took 0.02s
Build styles took 0.00s
Load screen analysis took 0.00s
Analyze screens took 0.03s
Save screen analysis took 0.02s
Prepare screens took 0.13s
Save pyanalysis. took 0.01s
Save bytecode. took 0.04s
Running _start took 0.00s

Full traceback:
  File "renpy/common/00start.rpy", line 211, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 388, in script
    $ __gl_test()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00gltest.rpy", line 388, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 340, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 2122, in interact
    self.start()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 983, in start
    self.post_init()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 1080, in post_init
    self.set_icon()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 1123, in set_icon
    with renpy.loader.load(icon, directory="images") as f:
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\loader.py", line 672, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'gui/window_icon.png'.

While running game code:
  File "renpy/common/00start.rpy", line 211, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 388, in script
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 388, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 340, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
OSError: Couldn't find file 'gui/window_icon.png'.

Full traceback:
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\main.py", line 635, in main
    run(restart)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\main.py", line 145, in run
    renpy.execution.run_context(True)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\execution.py", line 955, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 211, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 388, in script
    $ __gl_test()
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "renpy/common/00start.rpy", line 211, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 388, in script
    $ __gl_test()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00gltest.rpy", line 388, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 340, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 2122, in interact
    self.start()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 983, in start
    self.post_init()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 1080, in post_init
    self.set_icon()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 1123, in set_icon
    with renpy.loader.load(icon, directory="images") as f:
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\loader.py", line 672, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'gui/window_icon.png'.

While running game code:
  File "renpy/common/00start.rpy", line 211, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 388, in script
    $ __gl_test()
  File "renpy/common/00start.rpy", line 211, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 388, in script
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 388, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 340, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
OSError: Couldn't find file 'gui/window_icon.png'.
Traceback (most recent call last):
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\main.py", line 635, in main
    run(restart)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\main.py", line 145, in run
    renpy.execution.run_context(True)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\execution.py", line 955, in run_context
    context.run()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\execution.py", line 631, in run
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\execution.py", line 596, in run
    node.execute()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00gltest.rpy", line 388, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 340, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 2122, in interact
    self.start()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 983, in start
    self.post_init()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 1080, in post_init
    self.set_icon()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 1123, in set_icon
    with renpy.loader.load(icon, directory="images") as f:
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\loader.py", line 672, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'gui/window_icon.png'.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\renpy\editor.py", line 171, in launch_editor
    editor.end()
  File "C:\Users\meganschmid\Downloads\renpy-8.2.1-sdk\launcher\Visual Studio Code (System).edit.py", line 73, in end
    subprocess.Popen(args, creationflags=CREATE_NO_WINDOW)
  File "lib/python3.9/subprocess.py", line 951, in __init__
  File "lib/python3.9/subprocess.py", line 1420, in _execute_child
FileNotFoundError: [WinError 2] The system cannot find the file specified
